Build/build.ps1

```diff
@@ -323,63 +329,243 @@ function Update-RequiredModules {
     $moduleManifestUpdatedContent
 }
 
+<#
+    .Synopsis
+    Runs the unit tests of the specified module.
+
+    .Description
+    Runs the unit tests of the specified module. The tests are searched in a Tests\Unit path in the module directory.
+    The code coverage result of the tests gets updated in the ReadMe document. 
+
+    .Parameter ModuleName
+    Name of the module whose unit tests should be run
+#>
+function Start-UnitTests {
```

Comment:
Optional: Maybe Invoke-UnitTests could be a better name as you're using the **Invoke-Pester** cmdlet inside.